src/store/modules/stock/actions.ts

```diff
@@ -0,0 +1,42 @@
+import { StockService } from '@/services/StockService'
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import StockState from './StockState'
+import * as types from './mutation-types'
+import { hasError } from '@/utils'
+
+const actions: ActionTree<StockState, RootState> = {
+
+  /**
+   * Add stocks of specific product
+   */
+  async addProduct({ commit }, { productId }) {
+    const resp: any = await StockService.checkInventory({ productId });
+    if (resp.status === 200) {
+      commit(types.STOCK_ADD_PRODUCT, { productId, stock: resp.data.docs })
+    }
+  },
+
+  /**
+   * Add stocks of list of products
+   */
+  async addProducts({ commit }, { productIds }) {
```

Comment:
We have defined addProducts action to check the inventory but we have not called it and thus we will have 0 inventory for all the products.